xml/System.Runtime.InteropServices/MemoryMarshal.xml

```diff
@@ -661,7 +661,7 @@ This method allows a read-only memory buffer to be used in existing APIs that re
         <typeparam name="T">The type of the items in the read-only memory buffer.</typeparam>
         <typeparam name="TManager">The type of the <see cref="T:System.Buffers.MemoryManager`1" /> to retrieve.</typeparam>
         <param name="memory">The read-only memory buffer for which to get the memory manager.</param>
-        <param name="manager">When the method returns, the manager of <param name="memory" />. </param>
+        <param name="manager">When the method returns, the manager of <!-- incomplete sentence --><param name="memory" />. </param>
```

Comment:
Inside of these xml nodes, it should be `<paramref` not `<param` This whole XML file has a problem with this.